### ISSUE TITLE
Would love to see a CLI

### ISSUE THREAD

**Issue by fcisio** [josdejong/jsonrepair#34]
Hi, I'm generating json files with [Hygen JS](https://www.hygen.io/), and I'd love to be able to use this as a CLI.

Basically being able to run scripts like `> jsonrepair ./package.json`

Is that something that could happen?

**Comment by josdejong** [comment]
That's a great idea! 

I guess we should think through a few things of the API. It could work like:

```
$ jsonrepair data.json                         # Run a script file, output to console
$ jsonrepair data.json > repaired.json         # Run a script file, output to file
$ cat data.json | jsonrepair                   # Run input stream, output to console
$ cat data.json | jsonrepair > repaired.json   # Run input stream, output to file
```

What do you think would be the best behavior for the first syntax `jsonrepair data.json` ? Output to console, or replace the existing file `data.json`? I think the latter could be convenient but also dangerous.

**Comment by fcisio** [comment]
Hey, glad to hear you like the idea!

I was thinking about something like this:

```
$ jsonrepair data.json                        # Run a script file, output to console
$ jsonrepair data.json --write                # Run a script file, rewrite input file
$ jsonrepair data.json > repaired.json        # Run a script file, output to file
```

But I have a limited perspective on the project, since I only have that specific use case so far.
Feel free to go with what feel inlined with the project.

Thanks!

**Comment by josdejong** [comment]
Good idea with the `--write` option to specify that you want to overwrite. Maybe `--replace` or `--overwrite` is a more clear name for this.

**Comment by josdejong** [comment]
Francis, I've just published `v2.1.0` which has a command line interface, I hope you like it 😄 . Can you give it a try?

**Comment by fcisio** [comment]
Nice job 🎉
This works flawlessly on my end!
